Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -134,8 +134,9 @@ final class MethodModel: Model {
         assert(paramDecls.count == comps.count)
         
         let zippedParams = zip(paramDecls, comps)
-        self.params = zippedParams.map { (argAst: Structure, argLabel: String) -> ParamModel in
-            ParamModel(argAst, label: argLabel, offset: argAst.offset, length: argAst.length, data: data, isInitializer: ast.isInitializer)
+        self.params = zippedParams.map { (arg: (Structure, String)) -> ParamModel in
```

Comment:
You probably don't need this. Swift generally only gives this diagnostic when it needs it to give the real error.